packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -478,6 +488,7 @@ class _CircularProgressIndicatorState extends State<CircularProgressIndicator> w
         ),
         child: CustomPaint(
           painter: _CircularProgressIndicatorPainter(
+            backgroundColor: widget.backgroundColor,
```

Comment:
The logic here is to pipe the `backgroundColor` as originally sent to the widget, and then only paint it if it is not null.

I avoided assuming some default, like done in `LinearProgressIndicator`, to maintain the current behavior when no `backgroundColor` is given.

Reply:
Good, backwards compatibility is a priority!